File: Vorbereitung/Übungen/2/2-6.cs
namespace Uebung;

class Program
{
    static void Main()
    {
        int x = Console.ReadLine();
        Console.WriteLine($"Die {x}. Fibonacci Zahl ist {Fib(x)}.");
    }
    static int Fib(int x)
    {
        if (x == 1)
        {
            return 2;
        }
        if (x == 0)
        {
            return 1;
        }
        else
        {
            return x + x-1;
        }
    }
}

File: Vorbereitung/Übungen/2/2-6.c
int fib(int x) {
    if (x == 0) {
        return 1;
    }
    if (x == 1) {
        return 2;
    }
    else {
        return x + x-1;
    }
}

int main() {
    int a;
    scanf("%d", a);
    printf("Die %d. Fibonacci-Zahl ist %d", a, fib(a));
}

